windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -0,0 +1,71 @@
+== About {productwinc}
+
+{productwinc} enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+
+The release notes for {productwinc} track the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+ifndef::openshift-origin[]
+[id="getting-support"]
+== Getting support
+
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+{productwinc} is provided and available as an optional, installable component. {productwinc} is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for {productwinc}. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+
+For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
+
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
+endif::openshift-origin[]
+
+[id="wmco-7-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 7.0.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.0.0 were released in https://access.redhat.com/errata/RHSA-2022:9096.
+
+[id="wmco-7-0-0-new-features"]
+=== New features and improvements
+
+[id="wmco-7-0-0-wicd"]
+==== Windows Instance Config Daemon (WICD)
+
+The Windows Instance Config Daemon (WICD) is now performing many of the tasks that were previously performed by the Windows Machine Config Bootstrapper (WMCB). The WICD is installed on your Windows nodes. Users do not need to interact with the WICD and should not experience any difference in WMCO operation.
+
+[id="wmco-7-0-0-gcp"]
+==== Support for clusters running on Google Cloud Platform
+You can now run Windows Server 2022 nodes on a cluster installed on Google Cloud Platform (GCP). You can create a Windows `MachineSet` object on GCP to host Windows Server 2022 compute nodes. For more information, see ../windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc#creating-windows-machineset-gcp[Creating a Windows MachineSet object on vSphere].
```

Comment:
missing an xref keyword. Maybe this is intentional? Your .adoc filename appears in the preview.

Reply:
Intentional. We need to hide the WINC files when it releases after the associated OCP version (4.12 in this case).